cli/cmd/bb/bb.go

```diff
@@ -126,6 +126,9 @@ func run() (exitCode int, err error) {
 		return -1, err
 	}
 
+	// Show a picker if target argument is omitted.
+	args = picker.HandlePicker(args)
```

Comment:
no workflows implicitly depending on this, right? (we won't make anything start hanging?)

Reply:
Shouldn't change the behavior much (i.e. introduce pickers in any places where there previously weren't), we're just parsing this before splitting arguments before the `--` and after `--` so we can handle those "subtractive" patterns correctly (right now we sometimes add an extra `//...` argument where we shouldn't is all).